run_tests.py

```diff
@@ -27,6 +27,8 @@ def wait_until_rc_is_ready():
             rc_process = start_rc(rc_stdout, rc_stderr)
             try:
                 wait_until_rc_is_ready()
+                sys.stdout.flush()
```

Comment:
Magic...

I don't know. In the Windows runners, it was taking extra 5-7 minutes to set up the RC. While looking at the logs, I saw that Github Actions prints the logs of the script (`Downloading xxx` etc.) after the tests, where it should be before. So, I tried to flush the stdout before starting the tests and it solved the problem :smile:

Reply:
What about "--traverse-namespace",

Reply:
The documentation is not very clear. I read the explanation in nosetests docs.

Reply:
Oh, it is because our ancient nosetests dependency does not work with Python 3.8+ on Windows without it. https://stackoverflow.com/questions/58556183/nose-unittest-discovery-broken-on-python-3-8